scripts/bug_retriever.py

```diff
@@ -18,13 +18,12 @@ def retrieve_bugs(self, limit: int = None) -> None:
         bugzilla.set_token(get_secret("BUGZILLA_TOKEN"))
 
         last_modified = None
-        db.download(bugzilla.BUGS_DB)
-
-        # Get IDs of bugs changed since last run.
         try:
+            db.download(bugzilla.BUGS_DB)
```

Comment:
Handling two errors in one try-catch is not good practice. Also, I don't think we need to add `db.download()` in a try-catch. The issue mentions failure in `db.last_modified()`, not ``db.download()``.